BMM.Core/ViewModels/CuratedPlaylistsViewModel.cs

```diff
@@ -1,26 +1,61 @@
 ﻿using System.Collections.Generic;
+using System.Linq;
 using System.Threading.Tasks;
 using BMM.Api.Abstraction;
 using BMM.Api.Implementation.Models;
+using BMM.Core.Extensions;
+using BMM.Core.GuardedActions.Documents.Interfaces;
 using BMM.Core.Implementations.Caching;
-using BMM.Core.Implementations.DocumentFilters;
+using BMM.Core.Implementations.Languages;
+using BMM.Core.Implementations.Security;
 using BMM.Core.ViewModels.Base;
 
 namespace BMM.Core.ViewModels
 {
     public class CuratedPlaylistsViewModel : DocumentsViewModel
     {
+        private readonly IAppLanguageProvider _appLanguageProvider;
+        private readonly IUserStorage _userStorage;
+        private readonly ITranslateDocsAction _translateDocsAction;
+        private readonly IPrepareCoversCarouselItemsAction _prepareCoversCarouselItemsAction;
         public override CacheKeys? CacheKey => CacheKeys.PlaylistGetAll;
 
         public static readonly List<int> FeaturedIds = new List<int> { 31, 32, 33, 35, 36, 37 };
```

Comment:
this is not related to your PR. But these FeaturedIds is no longer needed. Can you remove it?
(Terrible naming by the way, since it doesn't tell me what it's for.)
As far as I understand, this was needed when we didn't have curated playlists yet. Instead we used a podcast and filtered it out of the list of podcast with these FeaturedIds.

Reply:
Ok, done